<HTML>
<HEAD>
 <TITLE> BayesPI2+</TITLE> 

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-55609707-2', 'auto');
ga('send', 'pageview');
</script>
</HEAD>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-H6G1XMQXV6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-H6G1XMQXV6');
</script>
<BODY BGCOLOR="FFFFF0"><b></b> 
<h2>
<a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-16-S7-S12">
Comprehensive genome-wide transcription factor analysis reveals that a combination of high affinity and low affinity DNA binding is needed for human gene regulation
<p></A></b> </h2> 
<br> 

<b><A HREF="mailto:junbai.wang@rr-research.no">Junbai Wang</a> 1</b>, Agnieszka Malecka 1, Gunhild Troen 1 and Jan Delabie 1 
<br><br> 
<b>
1. Department of Pathology<br>
<br>
Oslo University Hospital - The Norwegian Radium Hospital
<br>
Montebello 0310 Oslo
<br>
Norway

</b> <p> <br>
<b>Abstract
</b><br>
High-throughput in vivo protein-DNA interaction experiment is widely used in gene regulation study now. However, several challenges remain in analyzing the data such as distinguishing direct versus indirect protein-DNA interactions and identifying functional protein target genes. In particular, most computational methods only consider the top few hundreds or thousands strongest protein binding sites, where low binding affinity sites are completely ignored.
<br><br>
Here a new biophysical model of protein-DNA interactions (BayesPI2+) is developed to address the above-mentioned challenges. BayesPI2+ can be run in either a serial computation model or a parallel ensemble learning framework. It was successfully applied to distinguish direct versus indirect TF binding, by using either ChIP-seq data (equal-length genomic sequences; five human proteins) or ChIP-chip data (unequal-length genomic sequences; three yeast proteins). Subsequently, putative target genes of ESR1 and SPIB are investigated in detail in breast cancer cells and activated B cell-like diffuse large D-cell lymphoma cell lines, respectively. Both results reveal that direct binding requires contribution of indirect bindings to achieve sufficient binding affinity for initiating functional protein-DNA interaction. Finally, a verification of 10 putative direct SPIB binding sites by gel shift assay show 90% of them observed shift band.
<br><br>
These results suggest that differential binding affinity, obtained by applying BayesPI2+ on high-throughput in vivo protein-DNA binding data, is a good parameter to distinguish direct versus indirect protein binding. Further, functional target genes are regulated by both direct and indirect protein binding, a unique feature for identifying true protein target genes.
<br>
<br>
</br>
</body>
</html>
